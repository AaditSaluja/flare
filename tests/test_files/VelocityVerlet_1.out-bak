2022-01-27 16:09:48.190416
python 3.8.5 (default, Sep  4 2020, 02:22:02) 
[Clang 10.0.0 ]
numpy 1.20.3
ase 3.21.1
flare 0.2.4

Number of cpu cores: 1
Kernel: ['twobody', 'threebody']
Training points: 0
Cutoffs: {'twobody': 10.0, 'threebody': 6.0}
Number of hyperparameters: 5
Hyperparameter array: [0.64034029 0.16867265 0.0539972  0.4098916  0.05      ]
sig2: 0.64034029 
ls2: 0.16867265 
sig3: 0.0539972 
ls3: 0.4098916 
noise: 0.05 

Uncertainty tolerance: 1.0 times noise hyperparameter 
Timestep (ps): 0.001
Number of frames: 5
Number of atoms: 4
System species: {'H', 'He'}
Periodic cell (A): 
[[10.  0.  0.]
 [ 0.  5.  0.]
 [ 0.  0.  5.]]
Restart: 0 

Previous positions (A):
H        0.3112   -0.2531    0.2194
He       2.2071    2.4322    2.3684
H        5.2068    0.0473   -0.4642
He       7.6764    2.8988    2.1107
--------------------------------------------------------------------------------

Calling DFT...

DFT run complete.
Number of DFT calls: 1
Wall time from start: 0.02 s

*-Frame: 0 
Simulation Time:      0.000 ps 
El            Position (A)                    DFT Force (ev/A)                  Std. Dev (ev/A)                  Velocities (A/ps)       
H        0.3112   -0.2531    0.2194        0.0000    0.0000    0.0000        0.0000    0.0000    0.0000        0.0000    0.0000    0.0000
He       2.2071    2.4322    2.3684       -0.6846    0.0431   -0.0717        0.0000    0.0000    0.0000      -27.5032    8.4612   17.4141
H        5.2068    0.0473   -0.4642        0.2333   -0.1469   -0.1053        0.0000    0.0000    0.0000       18.8348   -2.2156   27.4965
He       7.6764    2.8988    2.1107        0.2457    0.3546   -0.1431        0.0000    0.0000    0.0000       20.2216    1.0676  -12.0402

Periodic cell (A): 
[[10.  0.  0.]
 [ 0.  5.  0.]
 [ 0.  0.  5.]]

Stress tensor (GPa):
       xx        yy        zz        yz        xz        xy
    10.863    10.879    10.890     0.071    -0.014    -0.099

Pressure (GPa): 10.877337 
Temperature (K): 1052.27 
Kinetic energy (eV): 0.408050 
Potential energy (eV): -9.330240 
Total energy (eV): -8.922191 

Wall time from start: 0.03 s
Adding atom [0, 1, 2, 3] to the training set.
Uncertainty: [0. 0. 0.]

GP hyperparameters: 
Hyp0 : sig2                           = 0.0000
Hyp1 : ls2                            = 1.8360
Hyp2 : sig3                           = 0.0012
Hyp3 : ls3                            = 0.5063
Hyp4 : noise                          = 0.0925
Likelihood: 5.9342
Likelihood gradient: [ 1.94587429e-03 -5.36581238e-09 -7.58131276e+02  3.56257707e+00
 -2.55329894e+00]
Wall time from start: 4.56 s
--------------------------------------------------------------------------------
-Frame: 1 
Simulation Time:      0.001 ps 
El            Position (A)                    GP Force (ev/A)                   Std. Dev (ev/A)                  Velocities (A/ps)       
H        0.3112   -0.2531    0.2194        0.0000    0.0000    0.0000        0.0989    0.0000    0.0000        0.0000    0.0000    0.0000
He       2.1789    2.4407    2.3858       -0.6179   -0.0006    0.0198        0.0925    0.0000    0.0000      -28.9777    8.4606   17.4558
H        5.2269    0.0443   -0.4369        0.2743   -0.1812   -0.0055        0.0537    0.0000    0.0000       21.4260   -3.9115   27.3188
He       7.6970    2.9002    2.0986        0.2835    0.2322   -0.0574        0.0754    0.0000    0.0000       20.8986    1.6326  -12.1685

Periodic cell (A): 
[[10.  0.  0.]
 [ 0.  5.  0.]
 [ 0.  0.  5.]]

Stress tensor (GPa):
       xx        yy        zz        yz        xz        xy
     1.209     1.090     1.263     0.216    -0.052    -0.216

Pressure (GPa): 1.187367 
Temperature (K): 1129.10 
Kinetic energy (eV): 0.437843 
Potential energy (eV): -0.124939 
Total energy (eV): 0.312904 

Wall time from start: 8.61 s

Calling DFT...

DFT run complete.
Number of DFT calls: 2
Wall time from start: 8.62 s

*-Frame: 1 
Simulation Time:      0.001 ps 
El            Position (A)                    DFT Force (ev/A)                  Std. Dev (ev/A)                  Velocities (A/ps)       
H        0.3112   -0.2531    0.2194        0.0000    0.0000    0.0000        0.0000    0.0000    0.0000        0.0000    0.0000    0.0000
He       2.1789    2.4407    2.3858       -0.7428    0.0501   -0.0677        0.0000    0.0000    0.0000      -28.9777    8.4606   17.4558
H        5.2269    0.0443   -0.4369        0.2734   -0.1541   -0.0792        0.0000    0.0000    0.0000       21.4260   -3.9115   27.3188
He       7.6970    2.9002    2.0986        0.2792    0.3509   -0.1579        0.0000    0.0000    0.0000       20.8986    1.6326  -12.1685

Periodic cell (A): 
[[10.  0.  0.]
 [ 0.  5.  0.]
 [ 0.  0.  5.]]

Stress tensor (GPa):
       xx        yy        zz        yz        xz        xy
    10.833    10.873    10.901     0.087    -0.001    -0.110

Pressure (GPa): 10.869201 
Temperature (K): 1129.10 
Kinetic energy (eV): 0.437843 
Potential energy (eV): -9.360297 
Total energy (eV): -8.922454 

Wall time from start: 8.62 s
Mean absolute errors & Mean absolute values
forces mae: 0.0490 eV/A
forces mav: 0.1796 eV/A
type 1 forces mae: 0.0170 eV/A
type 1 forces mav: 0.0844 eV/A
type 2 forces mae: 0.0811 eV/A
type 2 forces mav: 0.2748 eV/A
Adding atom [2, 3, 1, 0] to the training set.
Uncertainty: [0.05365265 0.         0.        ]

GP hyperparameters: 
Hyp0 : sig2                           = 0.0001
Hyp1 : ls2                            = 1.8360
Hyp2 : sig3                           = 0.0012
Hyp3 : ls3                            = 0.5596
Hyp4 : noise                          = 0.0774
Likelihood: 18.9087
Likelihood gradient: [ 4.23902291e-01 -4.47410568e-05  2.23580681e+01 -1.39830260e-02
 -1.09488873e+00]
Wall time from start: 9.29 s
--------------------------------------------------------------------------------
-Frame: 2 
Simulation Time:      0.002 ps 
El            Position (A)                    GP Force (ev/A)                   Std. Dev (ev/A)                  Velocities (A/ps)       
H        0.3112   -0.2531    0.2194        0.0000    0.0000    0.0000        0.0933    0.0000    0.0000        0.0000    0.0000    0.0000
He       2.1491    2.4491    2.4033       -0.6827   -0.0017    0.0251        0.0879    0.0000    0.0000      -30.6021    8.4574   17.5120
H        5.2498    0.0394   -0.4096        0.3124   -0.1987    0.0210        0.0496    0.0000    0.0000       24.3677   -5.7767   27.3938
He       7.7182    2.9021    2.0864        0.3186    0.2462   -0.0767        0.0736    0.0000    0.0000       21.6490    2.2304  -12.3416

Periodic cell (A): 
[[10.  0.  0.]
 [ 0.  5.  0.]
 [ 0.  0.  5.]]

Stress tensor (GPa):
       xx        yy        zz        yz        xz        xy
     1.514     1.356     1.574     0.264    -0.035    -0.222

Pressure (GPa): 1.481528 
Temperature (K): 1223.61 
Kinetic energy (eV): 0.474490 
Potential energy (eV): -0.173364 
Total energy (eV): 0.301125 

Wall time from start: 9.82 s

Calling DFT...

DFT run complete.
Number of DFT calls: 3
Wall time from start: 9.83 s

*-Frame: 2 
Simulation Time:      0.002 ps 
El            Position (A)                    DFT Force (ev/A)                  Std. Dev (ev/A)                  Velocities (A/ps)       
H        0.3112   -0.2531    0.2194        0.0000    0.0000    0.0000        0.0000    0.0000    0.0000        0.0000    0.0000    0.0000
He       2.1491    2.4491    2.4033       -0.8044    0.0569   -0.0630        0.0000    0.0000    0.0000      -30.6021    8.4574   17.5120
H        5.2498    0.0394   -0.4096        0.3188   -0.1625   -0.0530        0.0000    0.0000    0.0000       24.3677   -5.7767   27.3938
He       7.7182    2.9021    2.0864        0.3113    0.3465   -0.1708        0.0000    0.0000    0.0000       21.6490    2.2304  -12.3416

Periodic cell (A): 
[[10.  0.  0.]
 [ 0.  5.  0.]
 [ 0.  0.  5.]]

Stress tensor (GPa):
       xx        yy        zz        yz        xz        xy
    10.800    10.867    10.912     0.107     0.015    -0.124

Pressure (GPa): 10.859761 
Temperature (K): 1223.61 
Kinetic energy (eV): 0.474490 
Potential energy (eV): -9.397350 
Total energy (eV): -8.922861 

Wall time from start: 9.84 s
Mean absolute errors & Mean absolute values
forces mae: 0.0489 eV/A
forces mav: 0.1906 eV/A
type 1 forces mae: 0.0194 eV/A
type 1 forces mav: 0.0891 eV/A
type 2 forces mae: 0.0783 eV/A
type 2 forces mav: 0.2921 eV/A
Adding atom [2, 3, 1, 0] to the training set.
Uncertainty: [0.04959822 0.         0.        ]

GP hyperparameters: 
Hyp0 : sig2                           = 0.0023
Hyp1 : ls2                            = 1.8360
Hyp2 : sig3                           = 0.0011
Hyp3 : ls3                            = 0.5688
Hyp4 : noise                          = 0.0725
Likelihood: 32.9319
Likelihood gradient: [ 3.72137848e+00 -6.25049446e-02 -1.92964506e+02  7.30515724e-01
 -4.52604127e-01]
Wall time from start: 11.99 s
--------------------------------------------------------------------------------
-Frame: 3 
Simulation Time:      0.003 ps 
El            Position (A)                    GP Force (ev/A)                   Std. Dev (ev/A)                  Velocities (A/ps)       
H        0.3112   -0.2531    0.2194        0.0000    0.0000    0.0000        0.4260    0.0000    0.0000        0.0000    0.0000    0.0000
He       2.1176    2.4576    2.4209       -0.7555    0.0023    0.0232        0.4134    0.0000    0.0000      -32.3881    8.4616   17.5676
H        5.2757    0.0327   -0.3821        0.3456   -0.2117    0.0467        0.3981    0.0000    0.0000       27.6130   -7.7669   27.7174
He       7.7403    2.9046    2.0739        0.3747    0.2594   -0.0996        0.4091    0.0000    0.0000       22.5162    2.8570  -12.5673

Periodic cell (A): 
[[10.  0.  0.]
 [ 0.  5.  0.]
 [ 0.  0.  5.]]

Stress tensor (GPa):
       xx        yy        zz        yz        xz        xy
     1.820     1.525     1.765     0.279    -0.001    -0.233

Pressure (GPa): 1.703241 
Temperature (K): 1338.82 
Kinetic energy (eV): 0.519166 
Potential energy (eV): -0.050564 
Total energy (eV): 0.468602 

Wall time from start: 12.79 s

Calling DFT...

DFT run complete.
Number of DFT calls: 4
Wall time from start: 12.79 s

*-Frame: 3 
Simulation Time:      0.003 ps 
El            Position (A)                    DFT Force (ev/A)                  Std. Dev (ev/A)                  Velocities (A/ps)       
H        0.3112   -0.2531    0.2194        0.0000    0.0000    0.0000        0.0000    0.0000    0.0000        0.0000    0.0000    0.0000
He       2.1176    2.4576    2.4209       -0.8695    0.0632   -0.0577        0.0000    0.0000    0.0000      -32.3881    8.4616   17.5676
H        5.2757    0.0327   -0.3821        0.3701   -0.1721   -0.0267        0.0000    0.0000    0.0000       27.6130   -7.7669   27.7174
He       7.7403    2.9046    2.0739        0.3415    0.3408   -0.1813        0.0000    0.0000    0.0000       22.5162    2.8570  -12.5673

Periodic cell (A): 
[[10.  0.  0.]
 [ 0.  5.  0.]
 [ 0.  0.  5.]]

Stress tensor (GPa):
       xx        yy        zz        yz        xz        xy
    10.762    10.862    10.922     0.130     0.035    -0.142

Pressure (GPa): 10.848673 
Temperature (K): 1338.82 
Kinetic energy (eV): 0.519166 
Potential energy (eV): -9.442399 
Total energy (eV): -8.923232 

Wall time from start: 12.80 s
Mean absolute errors & Mean absolute values
forces mae: 0.0491 eV/A
forces mav: 0.2019 eV/A
type 1 forces mae: 0.0229 eV/A
type 1 forces mav: 0.0948 eV/A
type 2 forces mae: 0.0753 eV/A
type 2 forces mav: 0.3090 eV/A
Adding atom [2, 3, 1, 0] to the training set.
Uncertainty: [0.39813254 0.         0.        ]

GP hyperparameters: 
Hyp0 : sig2                           = 0.0044
Hyp1 : ls2                            = 1.8351
Hyp2 : sig3                           = 0.0010
Hyp3 : ls3                            = 0.5738
Hyp4 : noise                          = 0.0697
Likelihood: 47.6071
Likelihood gradient: [ -7.8582474   -0.23557939 -23.52146581   1.03002197  -2.64212921]
Wall time from start: 16.28 s
--------------------------------------------------------------------------------
-Frame: 4 
Simulation Time:      0.004 ps 
El            Position (A)                    GP Force (ev/A)                   Std. Dev (ev/A)                  Velocities (A/ps)       
H        0.3112   -0.2531    0.2194        0.0000    0.0000    0.0000        0.7893    0.0000    0.0000        0.0000    0.0000    0.0000
He       2.0843    2.4660    2.4384       -0.8289    0.0078    0.0197        0.7647    0.0000    0.0000      -34.3417    8.4776   17.6187
H        5.3051    0.0239   -0.3541        0.3893   -0.2194    0.0698        0.7458    0.0000    0.0000       31.2453   -9.8304   28.2687
He       7.7633    2.9078    2.0612        0.4254    0.2701   -0.1244        0.7623    0.0000    0.0000       23.4965    3.5061  -12.8507

Periodic cell (A): 
[[10.  0.  0.]
 [ 0.  5.  0.]
 [ 0.  0.  5.]]

Stress tensor (GPa):
       xx        yy        zz        yz        xz        xy
     1.982     1.697     1.934     0.289     0.052    -0.238

Pressure (GPa): 1.870924 
Temperature (K): 1477.69 
Kinetic energy (eV): 0.573020 
Potential energy (eV): 0.151059 
Total energy (eV): 0.724079 

Wall time from start: 17.28 s

Calling DFT...

DFT run complete.
Number of DFT calls: 5
Wall time from start: 17.28 s

*-Frame: 4 
Simulation Time:      0.004 ps 
El            Position (A)                    DFT Force (ev/A)                  Std. Dev (ev/A)                  Velocities (A/ps)       
H        0.3112   -0.2531    0.2194        0.0000    0.0000    0.0000        0.0000    0.0000    0.0000        0.0000    0.0000    0.0000
He       2.0843    2.4660    2.4384       -0.9404    0.0703   -0.0496        0.0000    0.0000    0.0000      -34.3417    8.4776   17.6187
H        5.3051    0.0239   -0.3541        0.4275   -0.1825   -0.0004        0.0000    0.0000    0.0000       31.2453   -9.8304   28.2687
He       7.7633    2.9078    2.0612        0.3717    0.3315   -0.1910        0.0000    0.0000    0.0000       23.4965    3.5061  -12.8507

Periodic cell (A): 
[[10.  0.  0.]
 [ 0.  5.  0.]
 [ 0.  0.  5.]]

Stress tensor (GPa):
       xx        yy        zz        yz        xz        xy
    10.711    10.851    10.924     0.151     0.068    -0.159

Pressure (GPa): 10.828549 
Temperature (K): 1477.69 
Kinetic energy (eV): 0.573020 
Potential energy (eV): -9.496823 
Total energy (eV): -8.923803 

Wall time from start: 17.29 s
Mean absolute errors & Mean absolute values
forces mae: 0.0475 eV/A
forces mav: 0.2138 eV/A
type 1 forces mae: 0.0242 eV/A
type 1 forces mav: 0.1017 eV/A
type 2 forces mae: 0.0708 eV/A
type 2 forces mav: 0.3258 eV/A
Adding atom [2, 3, 1, 0] to the training set.
Uncertainty: [0.74581451 0.         0.        ]

GP hyperparameters: 
Hyp0 : sig2                           = 0.0126
Hyp1 : ls2                            = 1.8349
Hyp2 : sig3                           = 0.0007
Hyp3 : ls3                            = 0.5755
Hyp4 : noise                          = 0.0667
Likelihood: 62.9569
Likelihood gradient: [-0.05863354 -0.80682615  0.46627342  0.23337914  0.06429358]
Wall time from start: 22.46 s
--------------------
Run complete.
