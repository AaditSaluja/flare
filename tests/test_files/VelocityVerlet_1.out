2022-01-27 16:12:50.738414
python 3.8.5 (default, Sep  4 2020, 02:22:02) 
[Clang 10.0.0 ]
numpy 1.20.3
ase 3.21.1
flare 0.2.4

Number of cpu cores: 1
Kernel: ['twobody', 'threebody']
Training points: 20
Cutoffs: {'twobody': 10.0, 'threebody': 6.0}
Number of hyperparameters: 5
Hyperparameter array: [1.26139003e-02 1.83487121e+00 6.75725290e-04 5.75460457e-01
 6.67258675e-02]
sig2: 0.012613900347414306 
ls2: 1.8348712118437158 
sig3: 0.0006757252903773871 
ls3: 0.5754604572110065 
noise: 0.06672586745932896 

Uncertainty tolerance: 1.0 times noise hyperparameter 
Timestep (ps): 0.001
Number of frames: 8
Number of atoms: 4
System species: {'H', 'He'}
Periodic cell (A): 
[[10.  0.  0.]
 [ 0.  5.  0.]
 [ 0.  0.  5.]]
Restart: 5 

Previous positions (A):
H        0.3112   -0.2531    0.2194
He       2.0489    2.4745    2.4561
H        5.3384    0.0130   -0.3255
He       7.7873    2.9116    2.0482
--------------------------------------------------------------------------------
--------------------------------------------------------------------------------
-Frame: 5 
Simulation Time:      0.005 ps 
El            Position (A)                    GP Force (ev/A)                   Std. Dev (ev/A)                  Velocities (A/ps)       
H        0.3112   -0.2531    0.2194        0.0000    0.0000    0.0000        0.0000    0.0000    0.0000        0.0000    0.0000    0.0000
He       2.0489    2.4745    2.4561       -0.9261    0.0096    0.0165        0.0000    0.0000    0.0000      -36.5120    8.4991   17.6620
H        5.3384    0.0130   -0.3255        0.4612   -0.2199    0.0977        0.0000    0.0000    0.0000       35.4956  -11.9017   29.0921
He       7.7873    2.9116    2.0482        0.4592    0.2780   -0.1534        0.0000    0.0000    0.0000       24.5610    4.1730  -13.2022

Periodic cell (A): 
[[10.  0.  0.]
 [ 0.  5.  0.]
 [ 0.  0.  5.]]

Stress tensor (GPa):
       xx        yy        zz        yz        xz        xy
     0.917     0.901     1.056     0.211     0.033    -0.163

Pressure (GPa): 0.958380 
Temperature (K): 1646.60 
Kinetic energy (eV): 0.638516 
Potential energy (eV): -0.524451 
Total energy (eV): 0.114065 

Wall time from start: 0.01 s
--------------------------------------------------------------------------------
-Frame: 6 
Simulation Time:      0.006 ps 
El            Position (A)                    GP Force (ev/A)                   Std. Dev (ev/A)                  Velocities (A/ps)       
H        0.3112   -0.2531    0.2194        0.0000    0.0000    0.0000        0.0000    0.0000    0.0000        0.0000    0.0000    0.0000
He       2.0112    2.4830    2.4738       -0.9733    0.0102    0.0122        0.0000    0.0000    0.0000      -38.8012    8.5230   17.6965
H        5.3761    0.0001   -0.2959        0.4955   -0.2256    0.1198        0.0000    0.0000    0.0000       40.0742  -14.0336   30.1329
He       7.8124    2.9162    2.0348        0.4910    0.2820   -0.1692        0.0000    0.0000    0.0000       25.7062    4.8479  -13.5910

Periodic cell (A): 
[[10.  0.  0.]
 [ 0.  5.  0.]
 [ 0.  0.  5.]]

Stress tensor (GPa):
       xx        yy        zz        yz        xz        xy
     1.007     1.008     1.152     0.222     0.046    -0.168

Pressure (GPa): 1.055804 
Temperature (K): 1841.67 
Kinetic energy (eV): 0.714161 
Potential energy (eV): -0.600070 
Total energy (eV): 0.114092 

Wall time from start: 0.02 s
--------------------------------------------------------------------------------
-Frame: 7 
Simulation Time:      0.007 ps 
El            Position (A)                    GP Force (ev/A)                   Std. Dev (ev/A)                  Velocities (A/ps)       
H        0.3112   -0.2531    0.2194        0.0000    0.0000    0.0000        0.0000    0.0000    0.0000        0.0000    0.0000    0.0000
He       1.9713    2.4916    2.4915       -1.0107    0.0067    0.0093        0.0000    0.0000    0.0000      -41.1925    8.5434   17.7224
H        5.4186   -0.0150   -0.2652        0.5282   -0.2292    0.1400        0.0000    0.0000    0.0000       44.9737  -16.2104   31.3764
He       7.8387    2.9213    2.0210        0.5133    0.2842   -0.1827        0.0000    0.0000    0.0000       26.9166    5.5304  -14.0152

Periodic cell (A): 
[[10.  0.  0.]
 [ 0.  5.  0.]
 [ 0.  0.  5.]]

Stress tensor (GPa):
       xx        yy        zz        yz        xz        xy
    -1.076    -0.238     0.172    -0.059    -1.248    -1.119

Pressure (GPa): -0.380594 
Temperature (K): 2064.08 
Kinetic energy (eV): 0.800410 
Potential energy (eV): -0.685862 
Total energy (eV): 0.114548 

Wall time from start: 0.39 s
--------------------
Run complete.
